validator/validator-main.protoascii

```diff
@@ -4880,7 +4880,7 @@ tags: {
   }
   attrs: {
     name: "src"
-    value_regex: "data:image\\/svg\\+xml;charset=utf-8,<svg height=\"\\d+(\\.\\d+)?\" width=\"\\d+(\\.\\d+)?\" xmlns=\"http:\\/\\/www\\.w3\\.org\\/2000\\/svg\" version=\"1\\.1\"\\/>|data:image\\/svg\\+xml;charset=utf-8,<svg height='\\d+(\\.\\d+)?\' width='\\d+(\\.\\d+)?\' xmlns='http:\\/\\/www\\.w3\\.org\\/2000\\/svg' version='1\\.1'\\/>"
+    value_regex: "data:image\\/svg\\+xml;charset=utf-8,\\s*<svg height=\"\\d+(\\.\\d+)?\" width=\"\\d+(\\.\\d+)?\" xmlns=\"http:\\/\\/www\\.w3\\.org\\/2000\\/svg\" version=\"1\\.1\"\\/>|data:image\\/svg\\+xml;charset=utf-8,\\s*<svg height='\\d+(\\.\\d+)?\' width='\\d+(\\.\\d+)?\' xmlns='http:\\/\\/www\\.w3\\.org\\/2000\\/svg' version='1\\.1'\\/>"
```

Comment:
Here's an alternative that would be more robust that would:

1. Allow any whitespace characters to precede the SVG element and between attributes (rather than just spaces).
2. Allow the attributes to appear in any order.
3. Allow the SVG tag to be self-closing or to have a closing tag.
4. Eliminate redundancy in the pattern by combining the single- and double-quoted variations. (Granted this makes the validated result slightly less robust, though it makes less prone to break in terms of maintenance.)

```suggestion
    value_regex: "data:image\\/svg\\+xml;charset=utf-8,\\s*<svg(\\s+height=[\"']\\d+(\\.\\d+)?[\"']|\\s+width=[\"']\\d+(\\.\\d+)?[\"']|\\s+xmlns=[\"']http:\\/\\/www\\.w3\\.org\\/2000\\/svg[\"']|\\s+version=\"1\\.1\"){4}\\s*(\\/>|<\\/svg>)"
```